### DISCUSSION TITLE
P/Invoke to System Informer possible?

### DISCUSSION THREAD

**Discussion by totalZero64** [winsiderss/systeminformer#1948] [Questions / Answers] [answered]
Is it possible to P/Invoke the exported phFunctions from C# by using classic p/invoke interop the same way we use Win32 API calls?
Or maybe use the provided sdk for plugins which states that it exports most of the ph Functions?

**Comment by dmex** [comment, marked as answer]
Yep, we used to have a C# wrapper back in 2010.
